ang/afform/afformSiteToken.aff.html

```diff
@@ -0,0 +1,13 @@
+<af-form ctrl="afform">
+  <af-entity data="{source: 'Site Token'}" type="SiteToken" name="SiteToken1" label="Site Token 1" actions="{create: true, update: true}" security="FBAC" url-autofill="1" />
+  <fieldset af-fieldset="SiteToken1" class="af-container">
+    <afblock-name-sitetoken></afblock-name-sitetoken>
+    <af-field name="name" defn="{required: true, input_attrs: {}}" />
```

Comment:
@colemanw or @totten : This form is to edit/create a site token, and I'd like the Name field to be read-only if `is_reserved = 1`. Any thoughts on how to do that in formBuilder?